### PR TITLE
Fix Responsiveness Issues

### LINKED ISSUE #55: Mobile Responsiveness
Currently, on mobile devices there seem to be some issues with responsiveness:
- The padding around the edge of the page is gone, causing cards to touch the edge of the page (looks nasty).
- On the resources page, some card titles go under the badge. It seems as if the large margin class isn't being applied or shrinks?
- The search and filters on the resources page get unaligned and messy. The search should have a line to itself with the filters beneath next to each other.

Tested on Apple iPhone XR with Safari.

**Please list any responsiveness issues you find on this thread.**

### PR DESCRIPTION
**Pull request by LamboCreeper**
Resolves #55 

#### Overview
- Removed the old hacky way of doing containers in favour of automatically setting margins
- Made the resources page's filter and searches responsive
- Stopped the titles of cards being hidden by the tags

### CODE REVIEW

File: src/css/pages/resources.css [hunk 1]
```diff
@@ -24,4 +24,16 @@
...
 
     font-size: 10px;
+}
+
+.resource-filters {
+    padding-bottom: 25px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-column-gap: 15px;
+    grid-row-gap: 15px;
```
Comment (LamboCreeper) [inline src/css/pages/resources.css:34]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903&bid=18631794)

File: src/css/pages/resources.css [hunk 2]
```diff
@@ -24,4 +24,16 @@
     margin-top: 25px;
 
     font-size: 10px;
+}
+
+.resource-filters {
+    padding-bottom: 25px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
```
Comment (LamboCreeper) [inline src/css/pages/resources.css:32]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/pages/resources.css [hunk 3]
```diff
@@ -24,4 +24,16 @@
...
+    grid-column-gap: 15px;
+    grid-row-gap: 15px;
+}
+
+.resource-filter-search {
+    grid-area: 1 / 1 / 2 / 3;
```
Comment (LamboCreeper) [inline src/css/pages/resources.css:38]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/pages/resources.css [hunk 4]
```diff
@@ -24,4 +24,16 @@
     margin-top: 25px;
 
     font-size: 10px;
+}
+
+.resource-filters {
+    padding-bottom: 25px;
```
Comment (LamboCreeper) [inline src/css/pages/resources.css:30]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/pages/resources.css [hunk 5]
```diff
@@ -24,4 +24,16 @@
...
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-column-gap: 15px;
```
Comment (LamboCreeper) [inline src/css/pages/resources.css:33]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/style.css [hunk 6]
```diff
@@ -96,13 +96,28 @@ body {
...
+    max-width: 1000px;
+    margin: 0 auto;
+}
 
-    display: flex;
-    justify-content: center;
+@media only screen and (max-width: 600px) {
+    .container {
+        padding: 15px;
+    }
```
Comment (LamboCreeper) [inline src/css/style.css:106]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903&bid=18631794)

File: src/css/pages/resources.css (hunk 1 above)
Comment (LamboCreeper) [inline src/css/pages/resources.css:34]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/pages/resources.css (hunk 2 above)
Comment (LamboCreeper) [inline src/css/pages/resources.css:32]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/pages/resources.css (hunk 3 above)
Comment (LamboCreeper) [inline src/css/pages/resources.css:38]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/pages/resources.css (hunk 4 above)
Comment (LamboCreeper) [inline src/css/pages/resources.css:30]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/pages/resources.css (hunk 5 above)
Comment (LamboCreeper) [inline src/css/pages/resources.css:33]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/style.css (hunk 6 above)
Comment (LamboCreeper) [inline src/css/style.css:106]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- src/components/molecules/Footer.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903&bid=18631794)

File: src/css/molecules/footer.css [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.footer-social-icons li a:hover {
+    color: var(--cs-blue);
+}
+
+.footer-copyright {
+    margin: 0;
+    text-align: center;
+    font-size: .5rem;
```
Comment (LamboCreeper) [inline src/css/molecules/footer.css:33]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

File: src/css/molecules/footer.css [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.footer-social-icons li {
+    display: inline;
+    font-size: 1.5rem;
+}
+
+.footer-social-icons li a {
+    color: var(--text);
+    transition: .1s;
```
Comment (LamboCreeper) [inline src/css/molecules/footer.css:23]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/components/molecules/Footer.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903&bid=18631794)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/components/molecules/Footer.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903&bid=18631794)

**Review by Jason2605 — APPROVED** [review]
LGTM 👍

**Pull request merged**